common/environment/build-style/go.sh

```diff
@@ -39,6 +39,7 @@ export CGO_CPPFLAGS="$CPPFLAGS"
 export CGO_CXXFLAGS="$CXXFLAGS"
 export CGO_LDFLAGS="$LDFLAGS"
 export CGO_ENABLED=1
+export GO111MODULE=auto
```

Comment:
If it's only a few templates, then wouldn't this be better per-template since `GOPATH` is deprecated?

Secondly, new packages shouldn't be using the old behavior.

Reply:
I don't think it's only a few, we have a lot of old Go stuff. But I don't want to deal with fixing them now. If you want to make a subsequent PR fixing them, I'd appreciate it, tho. I might even have an indirect list of packages that aren't using go modules, will have to check.